olive/olive_config.json

```diff
@@ -118,6 +118,10 @@
         "bnb": [ "bitsandbytes" ],
         "ort": [ "onnxruntime", "onnxruntime-directml", "onnxruntime-gpu", "onnxruntime-openvino", "numpy<2.0" ],
         "nvmo": [ "nvidia-modelopt~=0.11.0", "onnx-graphsurgeon" ],
-        "ort-genai": [ "onnxruntime-genai" ]
+        "ort-genai": [ "onnxruntime-genai" ],
+        "capture-onnx-graph": [ "onnxruntime-genai" ],
+        "cloud-cache": [ "azure-storage-blob" ],
+        "finetune": [ "onnxruntime-genai" ],
```

Comment:
can you add the other dependencies that come from `optimum` and `lora` also?

Reply:
Done